bin/releasenotes.sh

```diff
@@ -8,12 +8,12 @@
 # About:         Create a distro release notes markdown file based on plugin release notes
 #
 # Usage:         ./releasenotes.sh
-#                URLs of plugin release notes are taken from ./releasenotes.txt
+#                URLs of plugin release notes are taken from $RELEASENOTES_ORIGURL
```

Comment:
Does this mean the user has to define the RELEASENOTES_ORIGINAL environment variable for the script to work? What happens if that variable is undefined?

Consider changing this to a positional parameter, and add a useful error message if it is missing.

Reply:
Line 26 defined the variable already to the file releasenotes-orig-urls.txt.
Reason I write the variable name is because the variable name is static, while the file name can be changed.

Reply:
In that case, remove this comment. It is not relevant to the user and is confusing.

Reply:
It is relevant because user need to update that file will plugin release notes urls so the script will work.

Reply:
The $RELEASENOTES_ORIGURL variable is an implementation detail. It shouldn't be mentioned in the "usage" section.

It seems like the original comment was correct.

Reply:
Resolved